service/metadata.yaml

```diff
@@ -71,3 +71,89 @@ telemetry:
       gauge:
         async: true
         value_type: int
+
+    receiver.produced.items:
+      enabled: true
+      description: Number of items emitted from the receiver.
+      unit: "{item}"
+      sum:
+        value_type: int
+        monotonic: true
+    processor.consumed.items:
+      enabled: true
+      description: Number of items passed to the processor.
+      unit: "{item}"
+      sum:
+        value_type: int
+        monotonic: true
+    processor.produced.items:
```

Comment:
currently this metric is unused right? Will the processor in the graph have an outgoing prior to calling the next consumer?

Reply:
Good catch. I was using two layers of "consumed" instead of one each. Fixed.